app/controllers/alchemy/admin/pages_controller.rb

```diff
@@ -397,6 +397,14 @@ def serialized_page_tree
                                       user: current_alchemy_user,
                                       full: params[:full] == 'true')
       end
+
+      def application_controller_layout
+        layout_name = ApplicationController._layout
+        if layout_name && layout_name.is_a?(Proc)
+          layout_name = layout_name.call(ApplicationController.controller_path)
+        end
+        layout_name.presence || 'application'
+      end
```

Comment:
I would like to have a configuration option instead. Using a private API from Rails will give us trouble on each Rails update. Also this logic is pretty complex, while a simple config option is easy to add.